css/css-variables-1/variable-generated-content-dynamic-001-ref.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Test Reference</title>
+<style>
+div::before {
+  content: attr(data-foo);
+}
+</style>
+<div data-foo="PASS"></div>
```

Comment:
Does `<div>PASS</div>` not suffice here?

Reply:
It probably should, but blink has had problems with reftests that exercised text-shaping across different node boundaries (see https://github.com/w3c/csswg-test/pull/1143). That being said, that is being fixed, so I'll update, I guess.

Reply:
Just did that.